libs/components/layout/src/lib/modules/definition-list/definition-list.component.ts

```diff
@@ -32,5 +33,13 @@ export class SkyDefinitionListComponent {
     this.service.defaultValue.next(value);
   }
 
-  constructor(public service: SkyDefinitionListService) {}
+  constructor(public service: SkyDefinitionListService, logger: SkyLogService) {
+    logger.deprecated('SkyDefinitionListComponent', {
+      deprecationMajorVersion: 6,
+      moreInfoUrl:
+        'https://developer.blackbaud.com/skyux/components/description-list',
+      replacementRecommendation:
+        'We recommend `SkyDescriptionListComponent` instead.',
```

Comment:
I talked to John Lyons offline about this and he said he liked `Use x instead.` a bit better. I think we should put this sentence in the PR title (and consequently, the changelog), too.

Reply:
Updated. The PR title does now have periods - which isn't standard for our modern entries. But it doesn't make sense without them since we have two sentences.